dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/CommonTests/BasicBCLTests.java

```diff
@@ -888,9 +888,11 @@ public void BasicBCLTests_httpBackchannelLogoutUri_publicClient_httpsRequired_tr
         // logout expectations - just make sure we landed on the end_session/logout page - always with a good status code
         invokeLogout(webClient, updatedTestSettings, initLogoutWithHttpFailureExpectations(finalAppWithoutPostRedirect, client), response);
 
-        // Test uses the standard backchannelLogoutUri - so end_session with bcl steps should be performed - set expected states accordingly
+        // Access and refresh tokens should not be cleaned up since the BCL endpoint is not considered valid
         AfterLogoutStates states = new AfterLogoutStates(Constants.usesInvalidBCLEndpoint, updatedTestSettings.getFlowType(), logoutMethodTested, sessionLogoutEndpoint, updatedTestSettings.getRsTokenType());
-        //states.setIsRefreshTokenValid(true);
+        if (!currentRepeatAction.contains(Constants.END_SESSION)) {
+            states.setIsRefreshTokenValid(true);
+        }
 
```

Comment:
AfterLogoutStates should be setting accessTokenValid to false and validateLogoutResult will be making sure that the access_token is no longer valid.
So, either the comment on line 891 is wrong (and only the refresh token is NOT cleaned up) or validateLogoutResult should be failing in all non-END_SESSION cases)
(same comment for the other test cases that were updated)